source/command-line-options.txt

```diff
@@ -33,18 +35,74 @@ Use the following syntax to start |compass| and set a configuration option:
 Options
 -------
 
-You can set the following configuration options using the command line:
+You can set these options when using the command line:
+
+.. option:: --autoUpdates
+  
+   Enable :ref:`automatic updates <enable-automatic-updates>`.
+  
+.. option:: --enableFeedbackPanel
+  
+   Enable the feedback panel.
+   
+.. option:: --enableMaps
+ 
+   Enable geographic visualizations.
+
+.. option:: --exportConnections
+    
+   :ref:`Export <compass-export-connections-cli>` favorite connections.
+
+.. option:: --file <file>
+  
+   Specify a file containing a list of connections. These connections
```

Comment:
We don't have a page for the file itself, perhaps linking to the instructions for creating the file is a good alternative.

Specify a file containing a :ref:`list of connections <compass-export-connections-cli>`. These connections

Reply:
Good idea!